package.json

```diff
@@ -89,6 +89,7 @@
     ],
     "coveragePathIgnorePatterns": [
       "<rootDir>/build/",
+      "<rootDir>/default/src/pages/_document.jsx",
```

Comment:
removed since it is basic default code with only changes to head, but items in the head are tested in separate component tests.

Reply:
It shouldn't be ignored from code coverage